### ISSUE TITLE
@ApiProperty not working with swagger plugin on DTO classes

### ISSUE THREAD

**Issue by angel200sdnot** [nestjs/swagger#468]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When using the plugin `@nestjs/swagger/plugin` is not possible to override the api properties in a DTO class that is used with a `@Body` decorator in a controller. This causes the error `Uncaught SyntaxError: Unexpected identifier` when trying to use swagger ui

## Expected behavior

Being able to override the api properties of the class while using the plugin

## Minimal reproduction of the problem with instructions

Add  `@ApiProperty()` to one of the properties in the class `CreateCatDto` from the sample https://github.com/nestjs/nest/tree/master/sample/11-swagger

## What is the motivation / use case for changing the behavior?

Being able to be more descriptive on specific endpoints (description, examples, etc...)

## Environment

<pre><code>
Nest version:
- platform-express version : 6.10.14
- passport version         : 6.1.1
- swagger version          : 4.1.2
- typeorm version          : 6.2.0
- common version           : 6.10.14
- core version             : 6.10.14
- jwt version              : 6.1.1

 
For Tooling issues:
- OS Version     : Linux 5.0
- NodeJS Version : v10.18.0
- NPM Version    : 6.13.4

Others:

</code></pre>

**Comment by venkataramkishore** [comment]
I have the same problem unable to show swagger plugin using Swagger plugins
_   _             _      ___  _____  _____  _     _____ 
| \ | |           | |    |_  |/  ___|/  __ \| |   |_   _|
|  \| |  ___  ___ | |_     | |\ `--. | /  \/| |     | |  
| . ` | / _ \/ __|| __|    | | `--. \| |    | |     | |  
| |\  ||  __/\__ \| |_ /\__/ //\__/ /| \__/\| |_____| |_ 
\_| \_/ \___||___/ \__|\____/ \____/  \____/\_____/\___/ 


[System Information]
OS Version     : Windows 10
NodeJS Version : v12.13.1
NPM Version    : 6.12.1
[Nest Information]
platform-express version : 6.10.14
serve-static version     : 1.0.2
swagger version          : 4.1.2
typeorm version          : 6.2.0
common version           : 6.10.14
core version             : 6.10.14

PS D:\microservices\service> nest --version
6.13.1
in nest-clil.json i have added following:
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "src",
  "compilerOptions": {
    "plugins": ["@nestjs/swagger/plugin"]
  }
}

When using the plugin @nestjs/swagger/plugin is not possible to override the api properties in a DTO class that is used with a @Body decorator in a controller. This causes the error Uncaught SyntaxError: Unexpected identifier when trying to use swagger ui

**Comment by jeffminsungkim** [comment]
If I remove `@ApiProperty` decorator from a DTO class, the doc doesn't render parameters as well as Schemas.

```
Nest version:

"@nestjs/swagger": "^4.1.2",
"swagger-ui-express": "^4.1.2",
```

**Comment by kamilmysliwiec** [comment]
Fixed in 4.1.4

**Comment by jeffminsungkim** [comment]
@kamilmysliwiec Could you please re-open the issue? I've tested a few cases and the issue seems like still remains.

I've created a minimal [reproduction](https://github.com/jeffminsungkim/nesjs-swagger-v4-plugin-issue).

```
[System Information]
OS Version     : macOS Mojave
NodeJS Version : v11.15.0
NPM Version    : 6.13.3
[Nest Information]
platform-express version : 6.10.14
swagger version          : 4.1.5
swagger-ui-express: 4.1.2
common version           : 6.10.14
core version             : 6.10.14
```
